src/elektra/redshift.ini

```diff
@@ -0,0 +1,234 @@
+; Elektra specification file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; This file is based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
+;
+; A spec file "(...) defines how the configuration settings should be validated" (source: https://www.libelektra.org/tutorials/validate-configuration)
+; For details see: https://www.libelektra.org/tutorials/writing-a-specification-for-your-configuration)
+
```

Comment:
How will you support hooks? They are also in the config directory.

Reply:
Actually, thanks for pointing that out, I was unaware of hooks. 

Currently, any executable file that is in `~/.config/redshift/hooks` will be executed in certain state transitions by redshift (e.g. "daytime" -> "nighttime"). 

Hooks will be supported, even if I don't incorporate them in neither spec nor config file. 

One alternative I see is to only run hooks users have explicitly enabled by adding the hook name (or path) to some new configuration setting. But I believe this is a worse UX than the current situation.
Did you have something else in mind?

Reply:
Thanks, I understand now.
I did some research and discovered that @jonls has chosen against having hook scripts directly in the config (see https://github.com/jonls/redshift/pull/97#issuecomment-68199994), therefore I want to keep it like that.